Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -11,226 +11,17 @@ there is a manual procedure to get the quorum going again. The only requirement
-    --keyring=/etc/ceph/keyring-store/keyring \
-    --log-to-stderr=true \
-    --err-to-stderr=true \
-    --mon-cluster-log-to-stderr=true \
-    --log-stderr-prefix=debug \
-    --default-log-to-file=false \
-    --default-mon-cluster-log-to-file=false \
-    --mon-host=$ROOK_CEPH_MON_HOST \
-    --mon-initial-members=$ROOK_CEPH_MON_INITIAL_MEMBERS \
-    --id=b \
-    --setuser=ceph \
-    --setgroup=ceph \
-    --foreground \
-    --public-addr=10.100.13.242 \
-    --setuser-match-path=/var/lib/ceph/mon/ceph-b/store.db \
-    --public-bind-addr=$ROOK_POD_IP \
-    --inject-monmap=${monmap_path}
-```
-
-Exit the shell to continue.
-
-### Edit the Rook configmaps
-
-Edit the configmap that the operator uses to track the mons.
-
-```console
-kubectl -n rook-ceph edit configmap rook-ceph-mon-endpoints
-```
-
-In the `data` element you will see three mons such as the following (or more depending on your `moncount`):
-
-```yaml
-data: a=10.100.35.200:6789;b=10.100.13.242:6789;c=10.100.35.12:6789
-```
-
-Delete the bad mons from the list, for example to end up with a single good mon:
-
-```yaml
-data: b=10.100.13.242:6789
-```
-
-Save the file and exit.
-
-Now we need to adapt a Secret which is used for the mons and other components.
-The following `kubectl patch` command is an easy way to do that. In the end it patches the `rook-ceph-config` secret and updates the two key/value pairs `mon_host` and `mon_initial_members`.
-
-```console
-mon_host=$(kubectl -n rook-ceph get svc rook-ceph-mon-b -o jsonpath='{.spec.clusterIP}')
-kubectl -n rook-ceph patch secret rook-ceph-config -p '{"stringData": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "'"${good_mon_id}"'"}}'
-```
-
-!!! note
-    If you are using `hostNetwork: true`, you need to replace the `mon_host` var with the node IP the mon is pinned to (`nodeSelector`). This is because there is no `rook-ceph-mon-*` service created in that "mode".
-
-### Restart the mon
-
-You will need to "restart" the good mon pod with the original `ceph-mon` command to pick up the changes. For this run `kubectl replace` on the backup of the mon deployment yaml:
-
-```console
-kubectl replace --force -f rook-ceph-mon-b-deployment.yaml
-```
-
-!!! note
-    Option `--force` will delete the deployment and create a new one
-
-Start the rook [toolbox](ceph-toolbox.md) and verify the status of the cluster.
-
-```console
-ceph -s
-```
-
-The status should show one mon in quorum. If the status looks good, your cluster should be healthy again.
-
-### Restart the operator
-
-Start the rook operator again to resume monitoring the health of the cluster.
+If the name of the healthy mon is `c`, you would run the command:
 
 ```console
-# create the operator. it is safe to ignore the errors that a number of resources already exist.
-kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=1
+kubectl rook-ceph mons restore-quorum c
```

Comment:
```suggestion
kubectl rook-ceph mons restore-quorum c
```

can we add some snippet of output from the above command as an example? Also, I noticed we missed adding the output in the krew doc too. If you feel it is a good add, let's add output here and we can do krew in later PR.

Reply:
Actually I had a snippet ready to go and just realized I missed opening it. See #65.

Reply:
I'm assuming they'll link to the krew doc instead of needing to see the output here.

Reply:
yeah, since it was not added in krew earlier so I was suggesting to add here, but I think we are good now. Thanks!